staging/cse/windows/azurecnifunc.ps1

```diff
@@ -416,6 +417,36 @@ function New-ExternalHnsNetwork
         {
             $managementIP = $netIP.IPAddress
             $adapterName = $na.Name
+
+            Write-Log "Get node IPv4 address assigned to the adapter $($na.Name): $($managementIP)"
+            $nodeIPs += $managementIP

```

Comment:
just curious, would primary IP always the first one when secondary IPs are configured?

Reply:
I think that currently a AKS Windows VM does not have the second IP. So based on this, picking up the first or the last are same.

Reply:
> econdary IPs are configured?

@feiskyer Same question: Could we assign more than one IPv4 address in one physical NIC?

Reply:
the IPs are only configured on VM, but not configured in the NIC. So it would be OK here.